CHANGELOG/CHANGELOG-1.13.md

```diff
@@ -0,0 +1,89 @@
+- Add documentation for deploying Antrea with Rancher cluster. ([#4733](https://github.com/antrea-io/antrea/pull/4733), [@jainpulkit22])
+
+### Changed
+
+- Multiple APIs are promoted from alpha to beta.
+  - Promote ClusterGroup and Group to v1beta1. ([#5181](https://github.com/antrea-io/antrea/pull/5181), [@GraysonWu])
+  - Promote ExternalIPPool API to v1beta1. ([#5176](https://github.com/antrea-io/antrea/pull/5176), [@hongliangl])
+  - Promote Tier API to v1beta1. ([#5172](https://github.com/antrea-io/antrea/pull/5172), [@GraysonWu])
+  - Promote Egress API to v1beta1. ([#5180](https://github.com/antrea-io/antrea/pull/5180), [@wenqiq])
+  - Promote Upgrade AntreaClusterNetworkPolicy and AntreaNativeNetworkPolicy API to v1beta1. ([#5186](https://github.com/antrea-io/antrea/pull/5186), [@GraysonWu])
+  - Promote Traceflow API to v1beta1. ([#5108](https://github.com/antrea-io/antrea/pull/5108), [@luolanzone])
+- Add command `antctl upgrade api-storage` to support resource storage version migration. ([#5198](https://github.com/antrea-io/antrea/pull/5198), [@hongliangl])
+- Upgrade Open vSwitch to 2.17.7. ([#5225](https://github.com/antrea-io/antrea/pull/5225), [@antoninbas])
+- Upgrade Windows OVS version to 3.0.5. ([#5120](https://github.com/antrea-io/antrea/pull/5120), [@wenyingd])
+- Upgrade ClickHouse go client to v2. ([#5020](https://github.com/antrea-io/antrea/pull/5020), [@heanlan])
+- Remove Antrea Octant plugin. ([#5049](https://github.com/antrea-io/antrea/pull/5049), [@antoninbas])
+- Enhance L7 NetworkPolicy to support TLS protocol. ([#4932](https://github.com/antrea-io/antrea/pull/4932), [@hongliangl])
+- Remove the associated stale conntrack entries when UDP Endpoints are removed. ([#5112](https://github.com/antrea-io/antrea/pull/5112), [@hongliangl])
+- Revert a change to serve v1alpha2 version of the ClusterGroup CRD again for the consistent API promotion plan. ([#5277](https://github.com/antrea-io/antrea/pull/5277), [@GraysonWu])
+- Support DSR mode for Service's external addresses including LoadBalancerIPs and ExternalIPs on AntreaProxy, and document LoadBalancerMode related feature gate and configuration. ([#5202](https://github.com/antrea-io/antrea/pull/5202) [#5251](https://github.com/antrea-io/antrea/pull/5251), [@tnqn])
+- Enable `proxyAll` by default for AntreaProxy on Windows because the kube-proxy userspace datapath has been removed since kubernetes 1.26. ([#4980](https://github.com/antrea-io/antrea/pull/4980), [@XinShuYang])
+- Remove ClusterClaim API and upgrade ClusterSet version to v1alpha2, and enhance ClusterSet controller to support ClusterSet version upgrade. ([#5001](https://github.com/antrea-io/antrea/pull/5001) [#5250](https://github.com/antrea-io/antrea/pull/5250), [@luolanzone])
+- Increase Multi-cluster Controller's QPS for resource exchange and LabelIdentity worker count to improve performance. ([#5099](https://github.com/antrea-io/antrea/pull/5099), [@GraysonWu])
```

Comment:
Increase the controller QPS setting in Multi-cluster Controller to improve multi-cluster resource export performance, and increase the LabelIdentity controller worker count to improve its performance.